src/documentation/runners/apex.md

```diff
@@ -0,0 +1,8 @@
+---
+layout: default
+title: "Apache Apex Runner"
+permalink: /documentation/runners/apex/
+---
+# Using the Apache Apex Runner
+
+This page is under construction ([BEAM-825](https://issues.apache.org/jira/browse/BEAM-825)).
```

Comment:
In order to merge right away instead of waiting for `master`, I would like another sentence that says "this runner is on a feature branch".